fdbclient/vexillographer/fdb.options

```diff
@@ -57,6 +57,9 @@ description is not currently required but encouraged.
     <Option name="trace_file_identifier" code="36"
             paramType="String" paramDescription="The identifier that will be part of all trace file names"
             description="Once provided, this string will be used to replace the port/PID in the log file names." />
+    <Option name="trace_partial_file_suffix" code="39"
+            paramType="String" paramDesciption="Append this suffix to partially written log files. When a log file is complete, rename it to remove the suffix."
```

Comment:
```suggestion
            paramType="String" paramDesciption="Append this suffix to partially written log files. When a log file is complete, it is renamed to remove the suffix."
```

Reply:
We might also want to include a statement that no separator is used between the filename and this suffix (i.e. you would want to use ".ext" rather than "ext" in order to add the "ext" extension).

Reply:
Done